### PR TITLE
New event exporter config with support for new stackdriver resources

### PR DESCRIPTION
**Pull request by cezarygerard**
New event exporter, with support for use new and old stackdriver resource model.

This should also be cherry-picked to release-1.10 branch, as all  fluentd-gcp components support new and stackdriver resource model.

```release-note
Update event-exporter to version v0.2.0  that supports old (gke_container/gce_instance) and new (k8s_container/k8s_node/k8s_pod) stackdriver resources.
```

### CODE REVIEW

File: cluster/gce/gci/configure-helper.sh [hunk 1]
```diff
@@ -2153,12 +2153,8 @@ function update-prometheus-to-sd-parameters {
 
 # Updates parameters in yaml file for event-exporter configuration
 function update-event-exporter {
-  if [[ "${MULTIMASTER:-}" == "true" ]]; then
-    local -r location=${REGION:-}
-  else
-    local -r location=${ZONE:-}
-  fi
-  sed -i -e "s@{{ *event_exporter_location *}}@${location}@g" "$1"
+    stackdriver_resource_model="${LOGGING_STACKDRIVER_RESOURCE_TYPES:-old}"
```
[the remark this answers is not in the document]
Comment (cezarygerard) [reply cluster/gce/gci/configure-helper.sh]: New event exporter uses GCP metadata server for to figure out location for regional, zonal and standard clusters.

File: cluster/addons/fluentd-gcp/event-exporter.yaml [hunk 2]
```diff
@@ -42,15 +42,15 @@ spec:
...
     spec:
       serviceAccountName: event-exporter-sa
       containers:
       - name: event-exporter
-        image: k8s.gcr.io/event-exporter:v0.1.9
+        image: k8s.gcr.io/event-exporter:v0.2.0
         command:
         - /event-exporter
-        - -sink-opts=-location={{ event_exporter_location }}
+        - -sink-opts=-stackdriver-resource-model={{ exporter_sd_resource_model }}
```
Comment (MaciekPytel) [inline cluster/addons/fluentd-gcp/event-exporter.yaml]: It the location parameter no longer required? I understand why you add the new parameter, but it's not obvious why you're also removing the old one.

**Comment by MaciekPytel** [comment]
/approve 
For cluster/gce.

File: cluster/addons/fluentd-gcp/event-exporter.yaml [hunk 3]
```diff
@@ -42,15 +42,15 @@ spec:
...
       labels:
         k8s-app: event-exporter
-        version: v0.1.9
+        version: v0.2.0
     spec:
       serviceAccountName: event-exporter-sa
       containers:
       - name: event-exporter
-        image: k8s.gcr.io/event-exporter:v0.1.9
+        image: k8s.gcr.io/event-exporter:v0.2.1
```
Comment (x13n) [inline cluster/addons/fluentd-gcp/event-exporter.yaml]: If you're bumping this to 2.1, bump Deployment version as well.

**Comment by cezarygerard** [comment]
@x13n @MaciekPytel  can you approve and lgtm?

**Comment by MaciekPytel** [comment]
I already approved (changes remove lgtm, not approval). I'll leave proper review and lgtm to people with more context.

**Pull request merged**